src/framework/midi/midievent.h

```diff
@@ -104,14 +104,8 @@ struct Event {
             uint32_t full;
         } u;
         u.full = data;
-        if (
```

Comment:
This condition shall _obvioulsy_ check for MIDI messages that are in the range "Note off message" (0x8y) up to "Pitch bend change" (0xEy) and filter all System Exclusive/System Common messages (0xFy) that are proprietary to specific devices.

However, the check for `u.byte[0] & 0x80` is always true when the MSB (sign bit) is set, and so already makes the complete or'ed condition true. Line 109 to 114 are superflous as they are never reached.